deployment/Deploy.ps1

```diff
@@ -122,7 +122,7 @@ process {
         }
 
         $ActiveSubscription = Set-LtiActiveSubscription -Name $SubscriptionName -List $SubscriptionList
-        $UserEmailAddress = $ActiveSubscription.user.email
+        $UserEmailAddress = $ActiveSubscription.user.name
```

Comment:
why this change?

Reply:
Are we talking about the `$UserEmailAddress` or the `Set-LtiActiveSubscription` refactor?
- `$UserEmailAddress` was getting previously set as well but that was happening inside a `foreach` loop when we matched a subscription with a name. I simply separated out the 2 things -- matching subscription and using it to update appropriate variables.
-  The refactor was something which I did with a bit longer term goal of moving every part into its own module (like `Write-Log.ps1`). This should allow us to re-use these across `Deploy`, `Publish` and `Cleanup` scripts.

Reply:
the change I was talking about is user.`email` -> user.`name`

Reply:
I see this is something already present in the deploy script!